XCUITests/CronTabsPerfTests.swift

```diff
@@ -0,0 +1,243 @@
+import XCTest
+
+class CronTabsPerformanceTest: BaseTestCase {
+
+    let fixtures:[String:String] = ["testPerfTabs10": "tabsState10.archive","testPerfTabs20": "tabsState20.archive","testPerfTabs40": "tabsState40.archive","testPerfTabs80": "tabsState80.archive","testPerfTabs160": "tabsState160.archive","testPerfTabs320": "tabsState320.archive", "testPerfTabs640": "tabsState640.archive", "testPerfTabs1280": "tabsState1280.archive", "testPerfTabs20tabTray":"tabsState20.archive", "testPerfTabs1280tabTray":"tabsState1280.archive"]
+
+    override func setUp() {
+        // Test name looks like: "[Class testFunc]", parse out function name
+        let parts = name.replacingOccurrences(of: "]", with: "").split(separator: " ")
+        let functionName = String(parts[1])
+        let archiveName = fixtures[functionName]
+
+        // defaults
+        launchArguments = [LaunchArguments.PerformanceTest, LaunchArguments.SkipIntro, LaunchArguments.SkipWhatsNew, LaunchArguments.SkipETPCoverSheet]
+
+        // append specific load profiles to LaunchArguments
+        if fixtures.keys.contains(functionName) {
+            launchArguments.append(LaunchArguments.LoadTabsStateArchive + archiveName!)
+        }
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    // TODO: 1 perf test per tabsStateArchive of size:
+    // 1, 10, 20, 40, 80, 160, 320, etc.(tab count)
+    
+    // baseline
+//    func testPerfTabs1() {
```

Comment:
I guess we can remove this and set the baseline with 10 tabs? or if we set the baseline with one tab, this has to be uncommented